cookbooks/tutorial/doc/tutorial.tex

```diff
@@ -0,0 +1,29 @@
+\subsubsection{Running a geodynamic model}
+\label{sec:cookbooks-running-a-model}
+\textit{This section was contributed by Juliane Dannberg.}
+
+The input file for this model can be found at \url{cookbooks/tutorial/tutorial.prm}.
```

Comment:
Perhaps change the name to t`utorial_convection_box_2d.prm` if other parameter files will be added to the folder?

Reply:
I will let @jdannberg also comment, but I think we used `tutorial.prm` in a bunch of tutorials as the first file, and it would be nice to keep the name (so that old slides stll work). Also each parameter file will get its own folder, so as long as no other parameter file wants to claim the folder `tutorial` there should not be a conflict.

Reply:
That is a good point. I renamed it now to convection-box-particles, based on @tjhei's comments, but of course that means it's different in the slides now. But I could of course also update the slides.